Obvs.EventStore.Tests/TestServiceBus.cs

```diff
@@ -16,13 +17,14 @@ public class TestServiceBus
         public async void TestServiceBusWithLocalEventStore()
         {
             var serviceBus = ServiceBus.Configure()
-                .WithEventStoreEndpoints<ITestMessage1>()
-                    .Named("Obvs.EventStore.Test")
-                    .AppendMessageProperties(message => null)
-                    .FilterReceivedMessages(properties => true)
-                    .ConnectToEventStore("ConnectTo=tcp://admin:changeit@127.0.0.1:1113")
-                    .SerializedAsJson()
-                    .AsClientAndServer()
+                .WithEventStoreSharedConnectionScope("ConnectTo=tcp://admin:changeit@127.0.0.1:1113", config => config
+                    .WithEventStoreEndpoints<ITestMessage1>()
+                        .Named("Obvs.EventStore.Test")
+                        .AppendMessageProperties(message => null)
+                        .FilterReceivedMessages(properties => true)
+                        .ConnectToEventStore("ConnectTo=tcp://admin:changeit@127.0.0.1:1113")
```

Comment:
I did give it some thought, the options were:
- Change fluent config to allow ConnectToEventStore() to be optional, and throw if it isn't set
- Add another fluent config method called UsingSharedConnection() and throw if that's not set
- Duplicate the set of fluent config interfaces completely for use with shared connection config, and remove the ConnectToEventStore() method from the duplicated interfaces.

Thoughts?